### PR TITLE
the intellij documentation links in ide-tips-and-tricks dont actually work without the .html

### PR DESCRIPTION
**Pull request by AnAwesomGuy**
so i fixed them!

sorry for the pr spam :(

### CODE REVIEW

File: develop/ide-tips-and-tricks.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ Selecting some text and using <kbd>⌘/CTRL</kbd>+<kbd>Shift</kbd>+<kbd>↑ Up /
 In IntelliJ, the keybind for `Redo` may not be the usual <kbd>⌘/CTRL</kbd>+<kbd>Y</kbd> (Delete Line).
 Instead, it may be <kbd>⌘/CTRL</kbd>+<kbd>Shift</kbd>+<kbd>Z</kbd>. You can change this in the **Keymap**.
 
-For even more keyboard shortcuts, you can see [IntelliJ's documentation](https://www.jetbrains.com/help/idea/mastering-keyboard-shortcuts).
```
Comment (its-miroma) [inline develop/ide-tips-and-tricks.md:179]: It's redirecting automatically on my side

File: develop/ide-tips-and-tricks.md [hunk 2]
```diff
@@ -315,7 +315,7 @@ Anton Arhipov from Jetbrains also had an in depth talk about Regex Matching, Cod
 
 <VideoPlayer src="https://youtu.be/V8lss58zBPI" title="IntelliJ talk by Anton Arhipov on YouTube"/>
 
-For even more information, check out [Jetbrains' Tips & Tricks site](https://blog.jetbrains.com/idea/category/tips-tricks) and [IntelliJ's documentation](https://www.jetbrains.com/help/idea/getting-started).
```
Comment (its-miroma) [inline develop/ide-tips-and-tricks.md:318]: This also is redirecting automatically on my side

**Review by its-miroma — CHANGES_REQUESTED** [review]

File: develop/ide-tips-and-tricks.md (hunk 1 above)
Comment (its-miroma) [reply develop/ide-tips-and-tricks.md:179]: Actually, no it's not sorry

File: develop/ide-tips-and-tricks.md [hunk 3]
```diff
@@ -176,7 +176,7 @@ Selecting some text and using <kbd>⌘/CTRL</kbd>+<kbd>Shift</kbd>+<kbd>↑ Up /
 In IntelliJ, the keybind for `Redo` may not be the usual <kbd>⌘/CTRL</kbd>+<kbd>Y</kbd> (Delete Line).
 Instead, it may be <kbd>⌘/CTRL</kbd>+<kbd>Shift</kbd>+<kbd>Z</kbd>. You can change this in the **Keymap**.
 
-For even more keyboard shortcuts, you can see [IntelliJ's documentation](https://www.jetbrains.com/help/idea/mastering-keyboard-shortcuts).
+For even more keyboard shortcuts, you can see [IntelliJ's documentation](https://www.jetbrains.com/help/idea/mastering-keyboard-shortcuts.html).
```
Comment (its-miroma) [inline develop/ide-tips-and-tricks.md:180]:
```suggestion
<!-- markdownlint-disable-next-line search-replace -->
For even more keyboard shortcuts, you can see [IntelliJ's documentation](https://www.jetbrains.com/help/idea/mastering-keyboard-shortcuts.html).
```

**Review by its-miroma — CHANGES_REQUESTED** [review]

**Review by its-miroma — APPROVED** [review]
LGTM @IMB11

**Review by IMB11 — APPROVED** [review]

**Pull request merged**